Module.yaml

```diff
@@ -0,0 +1,74 @@
+version: 1.0.0
+name: OPCUAReader
+description: OPC UA Reader reads values for variables and forwards the payload to next module.
+icon: https://icons-020-demo.s3.eu-central-1.amazonaws.com/mi_OPCUA_input.png
+type: process
+tags:
+  - OPC UA Reader
+  - process
+categories:
+  - id: category
+    name: Process
+image:
+  name: weevenetwork/opcua-reader
+  tag: latest
+environments:
+  - key:  OPC_UA_SERVER
+    description: URL of OPC UA Server
+    value: ''
+    default: ''
+    type: string
+    name: OPC UA Server
+  - key:  REQUIRES_AUTHENTICATION
+    description: OPC UA Server requires authentication
+    value: ''
+    default: no
+    type: enum
+    options:
+      - yes
+      - no
+  - key:  OPC_UA_USERNAME
+    description: Username for authentication
+    value: ''
+    default: ''
+    type: string
+    name: OPC UA Username
+    dependencies:
+      - key: REQUIRES_AUTHENTICATION
+        value: yes
```

Comment:
dependencies field is not supported in V0 (we support in V1). This module is currently in V0 so please remove it.

Reply:
removed.